Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -647,9 +641,8 @@ static void getAllScriptsInPDFDocument(CGPDFDocumentRef pdfDocument, Vector<Reta
 {
     auto& document = element.document();
 
-#if ENABLE(UI_PROCESS_PDF_HUD)
     [m_pdfLayerController setDisplaysPDFHUDController:NO];
```

Comment:
Could you actually add a respondsToSelector check for this?  That will make it easier for PDFKit to remove old code.